### PR TITLE
Armorblox enhancement for more commands

### PR DESCRIPTION
**Pull request by SamhithaTatipalli**

### CODE REVIEW

**Comment by SamhithaTatipalli** [comment]
Hi @ostolero,

This PR is about code enhancement of existing pack Armorblox. We have added methods to support fetching an incident by its id, incident analysis, senders, actions, mail body and policies, update the Docker image and expose them as commands. 

Regards,
Samhitha.

**Comment by ostolero** [comment]
Hi @SamhithaTatipalli , thank you for the contribution, and great to see that your build is already green!
I have some other contributions to handle, so I might delay a bit more with this review.
I'll be sure to update here as soon as I'm done.
Thanks again!

**Comment by SamhithaTatipalli** [comment]
Sure, thanks @ostolero!

**Review by ostolero — CHANGES_REQUESTED** [review]
Hi @SamhithaTatipalli , the code looks great! the main things to fix are related to xsoar requirements more then code logic.
Note these changes-
1. All the commends are missing context info (both in yml and in the code) added comment it the python file with more info.
2. When you add the context info please update the readme again with the sdk method [generate-docs](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md) using a `command_exmaple` file to generate outputs.
3. Please add rn to describe your changes, this can be done with our [update-rn](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md) sdk command.
I'm here for any questions\clarifications or if you need help with any of the comments.
Thanks!

File: Packs/Armorblox/Integrations/Armorblox/Armorblox.py [hunk 1]
```diff
@@ -124,49 +135,80 @@ def get_remediation_action(client, incident_id):
...
-        if dt > int(start_time):
-            curr_incident = {'rawJSON': json.dumps(incident), 'details': json.dumps(incident)}
-            last_time = dt
-            incidents.append(curr_incident)
+        curr_incident = {'rawJSON': json.dumps(incident), 'details': json.dumps(incident)}
+        incidents.append(curr_incident)
     # Save the next_run as a dict with the start_time key to be stored
-    demisto.setLastRun({'start_time': str(last_time), 'pageToken': pageToken})
+    demisto.setLastRun({'last_fetch_time': last_fetch_time})
     return incidents
```
Comment (ostolero) [inline Packs/Armorblox/Integrations/Armorblox/Armorblox.py:169]: The `fetch_incidents` code looks way better! the reason for these changes is the client method that allowed to simplify this method?
I don't see you consider the `max_fetch` parameter, return only `max_fetch` results to load balance the system.

**Comment by ostolero** [comment]
Hi @SamhithaTatipalli , this pr has been quiet for a while, as I understand you are waiting for the client to proceed.
I'm closing this for now but open it when you get back to it and we can proceed. tnx!

**Pull request closed without merging**